fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,36 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+from subprocess import call, TimeoutExpired
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
+flags.DEFINE_enum('engine', 'libfuzzer', ['libfuzzer'],
+                  'the fuzzing engine used to do fuzzing test.')
+
+
+def main(argv):
+    if len(argv) < 2:
+        raise app.UsageError("Too few arguments")
+
+    ret_code = 0
+    cwd = os.getcwd()
+    try:
+        # Is this the right way to execute the runnable?
+        ret_code = call(cwd + "/../../../" + argv[1], timeout=FLAGS.timeout)
```

Comment:
Hmmmmmm i was under the impression that you wouldn't need cwd/../../../ and that just invoking the binary will work ok.